### ISSUE TITLE
Handle loading lazy attribute from printDefaultSizedImage

### ISSUE THREAD

**Issue by bic-ed** [bic-ed/zenphoto#8]
Do we really need lazy load for the main image in image.php? It apparently causes a performance downgrade. Perhaps we should either remove it or at least use a parameter to allow themes to skip the attribute. Something like:
`printDefaultSizedImage($alt, $class = null, $id = null, $title = null, $image = null, $loading = 'lazy')` 

Below the screenshots from two GTmetrics report, with and without the loading lazy attribute: 

### Loading lazy
![Summary Lazy](https://user-images.githubusercontent.com/2939831/199936816-184abb96-c82e-44c7-bdcc-d5ebeca7221b.png)
### No loading attribute
![Summary](https://user-images.githubusercontent.com/2939831/199936821-f9e5b6ca-6c32-4fa5-ad88-1700768e6aa8.png)

**Comment by acrylian** [comment]
You got some old tickets here. Yes, the first main image probably does not need lazyloading as this still applies. Sadly just based on the image function itself we do not really know if this is such a main or first image. All image "print" functions would all need an additional parameter to disable lazyloading on request.

**Comment by bic-ed** [comment]
Yes, these old tickets were here as reminders, I planned to discuss with the team before I had to abandon everything for a long while. I didn't even had memory of them. :-)
About this in particular, I guess the lazyload  can be removed at least from the printDefaultSizedImage() function, which is mainly used for the main image of image.php

**Comment by acrylian** [comment]
Probably most themes do use that as the main image. But you cannot be sure what custom stuff anyone really does…  And technically this also applies not only to the first main but also first images on a thumb list for example. I think a parameter is the most flexible.  I will think about that.

**Comment by acrylian** [comment]
Btw, Google Pagespeed is the better tool probably nowadays being even mobile first. Although all notes should be take as suggestions and not every noted possible problem is really a problem. Same applies to those Core Web VItals.

**Comment by bic-ed** [comment]
> I think a parameter is the most flexible. I will think about that.

I agree.
